service/frontend/workflow_handler.go

```diff
@@ -2720,7 +2722,50 @@ func (wh *WorkflowHandler) DescribeTaskQueue(ctx context.Context, request *workf
 		return nil, err
 	}
 
-	return matchingResponse.DescResponse, nil
+	resp := matchingResponse.DescResponse
```

Comment:
Do we have test coverage for this logic?

Reply:
I tested the relevant combinations manually. Do we need unit tests too? It's temporary code just to unstick this deployment

Reply:
Also not sure if unit test adds much value for this temporary code. We can do even more manual test in Cloud once the build is created.